### ISSUE TITLE
Keyboard navigation does not work if hideSelectColumn is true

### ISSUE THREAD

**Issue by stephanebarbey** [AllenFang/react-bootstrap-table#1150]
(Using 3.1.1:) If my table has the property hideSelectColumn, then the cell navigation with keyboard does not work any more, in that the tab key will sometime jump columns (usually from the first column to the third one).

I have included (bad quality) animated gifs that shows the behaviour. The only change between the two examples is the "hideSelectColumn" being commented out.

 ```
render() {
    const { data, columns, showFilter, selectionMode } = this.props;

    const selectRow = (selectionMode === 'none') ? {}
    : {
      mode: (selectionMode === 'single') ? 'radio' : 'checkbox',
      clickToSelect: true,
      hideSelectColumn: (selectionMode === 'single'),
      onSelect: this.onRowSelect,
      bgColor: 'orange'
    };

    const tableHeaderColumns = columns ? columns.map((col) => {
      return (
        <TableHeaderColumn
          id={col.key}
          key={col.key}
          dataField={col.key}
          dataSort
          dataFormat={col.dataFormat}
          isKey={col.isKey}
          hidden={col.hidden}
          filter={showFilter ? { type: 'TextFilter', placeholder: 'Please enter a value', delay: 500 } : null}
        >
          {col.name}
        </TableHeaderColumn>
      );
    }) : [];

    const cellEdit = {
      mode: 'click', // click cell to edit
      blurToSave: true,
    };

    return (
      <BootstrapTable
        ref={(c) => { this.table = c; }}
        data={data}
        striped
        condensed
        hover
        insertRow
        deleteRow
        search
        exportCSV
        bordered={Boolean(false)}
        keyBoardNav={Boolean(true)}
        selectRow={selectRow}
        cellEdit={cellEdit}
      >
        {tableHeaderColumns}
      </BootstrapTable>
    );
  }
```

Example without hideSelectColumn:

![workingexample](https://cloud.githubusercontent.com/assets/1199369/23947758/72840c80-0980-11e7-88d3-9a40d8a9cdc3.gif)

Example with hideSelectColumn:
![notworkingexample](https://cloud.githubusercontent.com/assets/1199369/23947764/7511a0ac-0980-11e7-82fa-122bd31e3b35.gif)

**Comment by AllenFang** [comment]
sorry for lately reply cause of my busy work.. I'll check it our and fix it as soon as possible, thanks

**Comment by AllenFang** [comment]
Fixed on `v3.1.6`, please have a look on it, thanks :)

**Comment by stephanebarbey** [comment]
I will! Thanks.
